cmd/notation/verify.go

```diff
@@ -45,16 +46,10 @@ func verifyCommand(opts *verifyOpts) *cobra.Command {
 Prerequisite: added a certificate into trust store and created a trust policy.
 
 Example - Verify a signature on an OCI artifact identified by a digest:
-  notation verify <registry>/<repository>@<digest>
+	notation verify <registry>/<repository>@<digest>
 
 Example - Verify a signature on an OCI artifact identified by a tag  (Notation will resolve tag to digest):
-  notation verify <registry>/<repository>:<tag>
-
-Example - [Experimental] Verify a signature on an OCI artifact referenced in an OCI layout using trust policy statement specified by scope.
-  notation verify --oci-layout <registry>/<repository>@<digest> --scope <trust_policy_scope>
-
-Example - [Experimental] Verify a signature on an OCI artifact identified by a tag and referenced in an OCI layout using trust policy statement specified by scope.
-  notation verify --oci-layout <registry>/<repository>:<tag> --scope <trust_policy_scope>
+	notation verify <registry>/<repository>:<tag>
```

Comment:
Are the indentations tabs or spaces? Please make sure that they are consistent across all commands.